server.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+require('dotenv').load();
+
+const http = require('http');
+const PORT = process.env.PORT;
```

Comment:
Not a point loser since Duncan is discouraging this, but I had to add an additional ``|| 3000`` in order for my tests to run properly because it wouldn't connect to the server. I don't think Duncan had you guys defaulting to env variables until after this lab passed. 

So just an FYI that since this is Lab 8-9, implementing current changes and techniques can cause mild problems, but it's still good to use them and be up to date. :)